tests/e2e/e2e.sh

```diff
@@ -17,83 +17,57 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
+ROOT=$(cd ../../../; pwd)
```

Comment:
I continue not to like this use of (cd ../../../; pwd).  There is an environment variable (IIRC) that contains `PWD` in the top level Makefile.  Can you see if that can be used?  If its not being passed to `Makefile.core.mk` - it can be easily added.